conf/ExcludeFromPackaging.php

```diff
@@ -24,9 +24,11 @@
         'vendor',
     ],
     'files' => [
+        'CODE_OF_CONDUCT.md',
         'DS_Store',
         'Dockerfile',
         'ExtensionBuilder.json',
+        'ExtensionBuilder.json',
```

Comment:
Why adding this a second time?

Reply:
Oh, this was my mistake when I resolved the merge conflict. I'll fix this in a minute. Good catch, thanks!

Reply:
#15